### PR TITLE
chore: moved types over from api skeleton

### PR DESCRIPTION
**Pull request by cajubelt**
plus some boilerplate to get this ready for publishing to NPM registry

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,274 @@
+
+/*
+/ Types for request and response bodies.
+/
+/ Typings for requests are grouped into three categories, each following a common naming convention
+/ that mimics the convention used by Express.
+/   - Request Query Parameters: Typed as `xxxRequestQuery`
+/   - Request Path Parameters: Typed as `xxxRequestParams`
+/   - Request Body: Typed as `xxxRequestBody`
```
Comment (jh2oman) [inline src/index.ts:9]: These names are a bit implementation specific to the API and the names are not as relevant to a developer using the helper library. I think this is fine as long as we re-export these types in the helper functions library with more relevant names. Like for the `getQuote` function the types could be `GetQuoteParams` and `GetQuoteResponse` etc.

**Review by jh2oman — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (jophish) [reply src/index.ts:9]: +1. These aren't types that will necessarily be exposed to end-users (i.e., developers using the FiatConnect client-side SDK); I imagine that we'll of course want to export some of these types verbatim through the SDK, but for others, (like these ones) we can export some type alias to make it more readable/even change the shape of them if we want (though I don't know if that's actually useful).

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
-# fiatconnect-types
\ No newline at end of file
+# fiatconnect-types
```
Comment (jophish) [inline README.md:1]: Maybe add a section here (primarily for our own benefit) about how to deploy/upload a new version of the package to NPM?

**Review by jophish — APPROVED** [review]
A'd! Looks good to me, though a README section about how to deploy a new version to NPM would be great.

**Pull request merged**